contribute/design/cstor-operator/day2ops.md

```diff
@@ -0,0 +1,309 @@
+### Introduction
+
+This document focuses on cStor day 2 operations workflow via the proposed specs
+of CSPC and CSP mentioned in contribute/design/cstor-operator/specs.md
+
+### Pool Expansion
+
+* “As a OpenEBS user, I should be able to add block devices to CSPC to increase
+   the pool capacity.” *
+
+Steps To Be Performed By User:
+
+1. `kubectl edit cspc your_cspc_name`
+
+2. Put the block devices in the CSPC spec against the correct kubernetes nodes.
```

Comment:
should we paste one BlockDevice as well.
Does BlockDevice contain the hostname i.e. correct kubernetes node